project.clj

```diff
@@ -23,4 +23,6 @@
              :clojure-1-6 {:dependencies
                            [[org.clojure/clojure "1.6.0"]]}
              :clojure-1-7 {:dependencies
-                           [[org.clojure/clojure "1.7.0"]]}})
+                           [[org.clojure/clojure "1.7.0"]]}
+             :clojure-1-8 {:dependencies
```

Comment:
Can you roll this profile into the `test-all` alias and verify it works?  And unless you have reason to support Clojure 1.5.1, you can update to drop that one while you're at it -- the earliest version I care about is 1.6.